vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -168,14 +168,14 @@ $global:map = @{
         "https://acs-mirror.azureedge.net/kubernetes/v1.21.14/windowszip/v1.21.14-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.22.1-hotfix.20211115/windowszip/v1.22.1-hotfix.20211115-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.22.4-hotfix.20220201/windowszip/v1.22.4-hotfix.20220201-1int.zip",
-        "https://acs-mirror.azureedge.net/kubernetes/v1.22.6-hotfix.20220130/windowszip/v1.22.6-hotfix.20220130-1int.zip",
```

Comment:
Please do not remove the old one since "v1.22.6-hotfix.20220130" is being used

Reply:
It is same for others

Reply:
fixed.